### DISCUSSION TITLE
F86D2949-C7F8-2D3E-CF22-15571E24319D

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#195] [General]
# Europa Clipper

Europa Clipper is a NASA interplanetary mission designed to study Europa in a series of 53 flybys while in orbit around Jupiter. Launched in 2024, it will arrive on Jupiter and its moons by 2030 and will study the system (alongside ESA's JUICE mission) until it is deliberately sent to crash onto Ganymede at the end of its mission in 2034

https://celestia.mobi/addon?item=F86D2949-C7F8-2D3E-CF22-15571E24319D

**Comment by WasiAdiyatHasan** [comment]
Where is Europa Clipper It isn't working

**Reply by DaveBowman2001** [reply]
Europa Clipper wasn't launched yet. You need to set your date beyond October 2024 to see it in Celestia

**Reply by DaveBowman2001** [reply]
I have now updated the (postlaunch) trajectory up to 2026. Clipper should begin appearing on Celestia by October 14th (actual launch day)
